Automattic/jetpack
# I18n: Move GlotPress locales file into compat package

Fixes #23151.

#### Changes proposed in this Pull Request:

* Moves `locales.php` (which defines `GP_Locale`/`GP_Locales`) from the Jetpack plugin into the `compat` package.
* Uses `GP_Locales` from the `compat` package in the `search` package.





#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Apply this change to your connected Jetpack site.
* Ensure that usage of `GP_Locale` or `GP_Locales` in the Jetpack plugin continues to work as expected. Places to test: [Adminbar Notification menu](https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/modules/notes.php#L199), [Facebook sharing JS payload URL](https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php#L1233), [follow button](https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/modules/widgets/follow-button.php#L82).
